styles/widgets/generic/common.generic.less

```diff
@@ -2,56 +2,58 @@
     font-family: "dx.generic@{GENERIC_COLOR_POSTFIX}@{GENERIC_SIZE_POSTFIX}";
 }
 
-.dx-theme-accent-as-text-color {
-    color: @base-accent;
-}
-
-.dx-theme-accent-as-background-color {
-    background-color: @base-accent;
-    fill: @base-accent;
-}
-
-.dx-theme-accent-as-border-color {
-    border-color: @base-accent;
-}
-
-.dx-theme-text-color {
-    color: @base-text-color;
-}
-
-.dx-theme-text-color-as-background-color {
-    background-color: @base-text-color;
-    fill: @base-text-color;
-}
-
-.dx-theme-text-color-as-border-color {
-    border-color: @base-text-color;
-}
-
-.dx-theme-background-color {
-    background-color: @base-bg;
-    fill: @base-bg;
-}
-
-.dx-theme-background-color-as-text-color {
-    color: @base-bg;
-}
+body.dx-theme-generic {
```

Comment:
body.dx-theme-generic -> .dx-theme-generic?

Reply:
The idea is to make the cascade stronger. We always have body so...

Reply:
But changing body to .dx-viewport would be smart. I'll do it.